daemon/network.go

```diff
@@ -574,7 +574,7 @@ func (daemon *Daemon) GetNetworks(filter filters.Args, config types.NetworkListC
 
 	if config.Detailed {
 		for i, n := range list {
-			np := &n
+			np := &list[i]
 			buildDetailedNetworkResources(np, idx[n.ID], config.Verbose)
```

Comment:
Is it worth switching `n.ID` here to `np.ID` and ditching `n` completely?

Reply:
Yes, makes sense; updated 👍